_episodes_rmd/03-loops-R.Rmd

```diff
@@ -45,7 +45,7 @@ analyze("data/inflammation-01.csv")
 
 We can use it to analyze other data sets one by one:
 
-```{r inflammation-02}
+```{r inflammation-02, fig. alt = "A scatterplot of minimum daily inflammation over 40 days, increasing in a step-wise pattern until day 20, and then decreasing in a step-wise pattern to visualize inflammation levels in a single dataset."}
```

Comment:
```suggestion
```{r inflammation-02, fig.alt = "A scatterplot of minimum daily inflammation over 40 days, increasing in a step-wise pattern until day 20, and then decreasing in a step-wise pattern to visualize inflammation levels in a single dataset."}
```